packages/forma-36-website/src/content/getting-started.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: 'Getting started'
+---
+
+## Installation
+
+### Install components
+
+Inside your React project directory, install Forma 36 by running either of the following:
+
+**When using NPM:**
+
+```bash static=true
+npm i @contentful/f36-components
+```
+
+**When using Yarn:**
+
+```bash static=true
+yarn add @contentful/f36-components`
+```
+
+#### Installing components as separate packages
+
+ou can install Forma 36 React components as separate packages if that's what you need for your project. For example, here's how to install just the button:
```

Comment:
missing first letter — "You can install"

Reply:
oops. fixed

Reply:
[Suggestion]

Merge the two paragraphs into:
"Although we recommend using `@contentful/f36-components`, you can install Forma 36 React components as separate packages if that's what you need for your project. This is called tree shaking and it will make your bundle contain only components that you actually import and need. For example, here's how to install just the button:"